rm-ethereum.conf

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can super access these values in JSON file with the library? Because some of these values in the config file are the same with JSON files.

Reply:
Yes, we will do it after the demo.